demo/Main.elm

```diff
@@ -23,20 +22,18 @@ main =
 -- MODEL
 
 
-type alias Data =
-    Service.ListQueuesResult
-
-
 type alias Model =
-    { response : Maybe (AWS.Response Data)
+    { queueUrls : List String
     , err : Maybe Http.Error
     }
 
 
 init : ( Model, Cmd Msg )
 init =
-    ( Model Nothing Nothing
-    , Task.perform GotDate Date.now
+    ( Model [] Nothing
+    , SQS.listQueues AWS.defaultOptions
+        |> AWS.toTask (SQS.config Config.region) creds
+        |> Task.attempt ListQueues
```

Comment:
I think `toTask` makes sense. It's explicit in that it's asynchronous from the "callers" perspective and keeps the terminology inline with Elm's.